### PR TITLE
Laura & Jessica 

### PR DESCRIPTION
**Pull request by jessand77**
https://movies-playing-now-project.netlify.app

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
+import MoviePage from 'components/MoviePage'
+
+
 export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
+  return <MoviePage />
```
Comment (rijad90) [inline code/src/App.js:7]: You remebered that you don't need a "<> </>" when its only 1 component, nice!

File: code/src/components/MovieDetails.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+            font-size: 19px;
+        }
+        p {
+            font-size: 17px;
+        }
+    }
+    @media (min-width: 668px){
+        flex-direction: column;
+        align-self: flex-end;
+    }
```
Comment (rijad90) [inline code/src/components/MovieDetails.js:84]: Styled components looks good, very good :)

File: code/src/components/MovieDetails.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+        navigate(-1);
+    }
+
+    useEffect(() => {
+        fetch(MOVIE_DETAIL_URL(id))
+            .then((res) => res.json())
+            .then((data) => setMovie(data))
+    }, [id])
+
+    console.log(movie)
```
Comment (rijad90) [inline code/src/components/MovieDetails.js:103]: dont forget to remove or comment out the console logs

File: code/src/utils/urls.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+const API_KEY = '6fb1d64d540188673e57f0b42d2bacb8'
+
+export const MOVIE_LIST_URL = `https://api.themoviedb.org/3/movie/now_playing?api_key=${API_KEY}&language=en-US&page=1`
+
+export const POSTER_URL = `https://image.tmdb.org/t/p/w342`
+
+export const BACKDROP_URL = `https://image.tmdb.org/t/p/w1280`
+
+export const MOVIE_DETAIL_URL = (movieId) => `https://api.themoviedb.org/3/movie/${movieId}?api_key=${API_KEY}&language=en-US`
```
Comment (rijad90) [inline code/src/utils/urls.js:9]: creating a comonents for the api calls was a good choice

**Review by rijad90 — APPROVED** [review]
Good job this week, you created multiple pages using routes which was this weeks task! The structure of code was good and easy to read. The only details I was wondering about is why you did not create media quary for mobile, I have seen your other projects so I know you can do it, still it was not necessary to create it this week so I congratulate you on a good week, well done! :)

File: code/src/App.js (hunk 1 above)
Comment (EmmaaBerg) [reply code/src/App.js:7]: Nice that you kept the entry point clean!

File: code/src/components/MovieDetails.js [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+             
+    `
+
+const MovieDetails = () => {
+    const [movie, setMovie] = useState({})
+    const { id } = useParams()
+    const navigate = useNavigate()
+
+    const onBackButtonClick = () => {
+        navigate(-1);
```
Comment (EmmaaBerg) [inline code/src/components/MovieDetails.js:94]: Good job with the navigate for your button

File: code/src/index.css [hunk 6]
```diff
@@ -11,3 +11,42 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
```
Comment (EmmaaBerg) [inline code/src/index.css:14]: Good job with using styled components for your project and that the CSS here is for more global things :D

File: code/src/utils/urls.js (hunk 4 above)
Comment (EmmaaBerg) [reply code/src/utils/urls.js:9]: Agree with Rijad, good to have all the URLs on one page and good that you stored the key in a separate variable.

**Review by EmmaaBerg — COMMENTED** [review]
Well done with this project. The components were well structured and the code was easy to follow and great job with the styled components! One detail I noticed was when I look at your page on my computer the go-back button was hidden behind the 'poster and text area', don't know if it is just on my screen but if you have time later on maybe you would like to double check. But overall, really well done! :D

**Pull request closed without merging**